### PR TITLE
Updated scenario to use 3 zookeeper instances.

### PR DESCRIPTION
**Pull request by JumaX**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Updated plaintext-rhel scenario to use 3 zookeeper instances for future upgrade testing.

Have added the following additional scenarios for more robust upgrade testing:

mtls-customcerts-rhel
mtls-debian.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

All scenarios have been run with the verification option to confirm they work.

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

**Comment by JumaX** [comment]
@domenicbove Added additional scenarios, suspect they will lead to merge conflict due to changes required for the older version of CP-Ansible.  Should not be hard to resolve.

File: roles/confluent.test/molecule/Dockerfile-debian.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    apt-get install --yes \
+      apt-transport-https \
+      gnupg \
+      software-properties-common \
+      openjdk-8-jdk \
+      rsync \
+      ca-certificates \
+      openssl \
+      unzip \
+      curl
```
Comment (domenicbove) [inline roles/confluent.test/molecule/Dockerfile-debian.j2:13]: Is this dockerfile and the other one missing the confluent packages?

Comment (JumaX) [reply roles/confluent.test/molecule/Dockerfile-debian.j2:13]: Yep, it just installs what is missing to get the scenario working as the images don't contain all required packages.

Comment (domenicbove) [reply roles/confluent.test/molecule/Dockerfile-debian.j2:13]: yep i remember now, i think in the 600 branch (or 550) i improved the common role so this is not necessary, but it def is in 540

**Review by domenicbove — APPROVED** [review]
LGTM

**Pull request merged**